release/models/platform/openconfig-platform.yang

```diff
@@ -534,6 +541,89 @@ module openconfig-platform {
     }
   }
 
+  grouping platform-health-top {
+    description
+      "Grouping containing health-related parameters.";
+
+    container health {
+      description
+        "The health of the component. The paramaters within this
+        container indicate the status of the component beyond whether
+        it is operationally up or down. When a signal is received
```

Comment:
Am I reading this correctly, that the only difference the `health` status conainer provides right now when compared to oper-status=up/down is the new state called "Degraded", which can indicate a partially operational status?

Reply:
I'd imagine that it is possible to have such a case in systems that don't take auto-remediation of a particular case. For example, let's consider a disk that is full -- potentially, we might have `oper-status = ACTIVE` (given that we might not be able to offline such a disk if it was mounted as `/` for example), but 100% usage which would give us `status = UNHEALTHY`.